css/css-contain/contain-style-counters-004.html

```diff
@@ -63,9 +63,11 @@
   <p> <span></span> <span></span> <span></span> <span></span>
 
   <!-- 4 span inside the <p>. However, since div isn't a sibling of spans,
-       it creates a new counter.
+       it creates a new counter, which is allowed to access the 13 from the
```

Comment:
The GH UI for suggestions is broken so I can't actually drop a suggestion here, but the text should instead say:

```
However, since div isn't a sibling of the spans,
it sees the original counter value on body, as per example at:
```